tools/read-conf.py

```diff
@@ -84,6 +140,7 @@ def print_keyval(result, args):
 if __name__ == "__main__":
     parser = argparse.ArgumentParser(description='Load config, print stuff')
     parser.add_argument('-c', '--config', default='config.conf', help='Config file name (default: config.conf)')
+    parser.add_argument('-C', '--custom', default='custom.conf', help='Customization file name (default: custom.conf)')
```

Comment:
@sushain97 What about `internal-constants.conf`? (@Androbin don't forget to rename variables once we decide.)

Reply:
But "internal" sounds like the path should not be configurable at all.

Reply:
What's wrong with `constants.conf`? Seems OK to me.

Reply:
Well, `config.conf` contains "constants" as well.

Reply:
Fair enough. Hmm... `advanced-config.conf`?